spec/lib/wits/five_minute_prices_spec.rb

```diff
@@ -52,8 +52,8 @@
           expect(response[:date]).to            eq date
           expect(response[:prices].length).to   be_within(2).of(48)
 
-          expect(price[:time]).to           eq parse_nz_time(Time.parse("#{date} 03:30"))
-          expect(price[:trading_period]).to eq 8
+          expect(price[:time]).to           eq parse_nz_time(Time.parse("#{date} 01:30"))
```

Comment:
Line is too long. [89/80]<br>Unnecessary spacing detected.<br>Put one space between the method name and the first argument.